features/homepage.feature

```diff
@@ -0,0 +1,34 @@
+Feature: The homepage
```

Comment:
Nice - I've not seen someone actually implement these before! These tests failed out of the box for me, but worked when I ran them in a Docker container - are you running them in WSL by any chance? If so I think it's worth us making a note in your docs that they may not work on Windows, but I'm happy here for us to just to accept that
If not, I'm not sure what caused that error for me